packages/jellyfish-api-core/__tests__/category/poolpair/testpoolswap.test.ts

```diff
@@ -57,6 +63,9 @@ describe('Poolpair', () => {
       tokenTo: 'DFI'
     }) // 199.99999729@0
 
+    if (typeof result !== 'string') {
+      throw new Error(`result was ${typeof result}, expected string`)
+    }
```

Comment:
Since this is a test, I think a more appropriate way of doing this is:

```suggestion
    expect(typeof result).toStrictEqual('string')
```